src/frontend/src/pages/FinancePage/ReimbursementRequestsSection.tsx

```diff
@@ -93,40 +98,34 @@ const ReimbursementRequestTable = ({
 
   const rows = displayedReimbursementRequests.map(createReimbursementRequestRowData).sort((a, b) => {
     if (orderBy === 'vendor') {
-      return order === 'desc'
+      return !isAscendingOrder
         ? vendorDescendingComparator(a.vendor, b.vendor)
         : -vendorDescendingComparator(a.vendor, b.vendor);
     }
     if (orderBy === 'status') {
-      return order === 'desc'
+      return !isAscendingOrder
         ? statusDescendingComparator(a.status, b.status)
         : -statusDescendingComparator(a.status, b.status);
     }
     if (orderBy === 'submitter') {
-      return order === 'desc'
+      return !isAscendingOrder
         ? submitterDescendingComparator(a.submitter, b.submitter)
         : -submitterDescendingComparator(a.submitter, b.submitter);
     }
     if (b[orderBy] === undefined) {
       return -1;
     }
-    return order === 'desc' ? descendingComparator(a, b, orderBy) : -descendingComparator(a, b, orderBy);
+    return !isAscendingOrder ? descendingComparator(a, b, orderBy) : -descendingComparator(a, b, orderBy);
   });
 
   const tabs = [{ label: 'My Requests', value: 0 }];
   if (canViewAllReimbursementRequests) tabs.push({ label: 'All Club Requests', value: 1 });
 
   const handleRequestSort = (property: keyof ReimbursementRequestRow) => {
-    const isAsc = orderBy === property && order === 'asc';
-    setOrder(isAsc ? SortingOrder.descending : SortingOrder.ascending);
+    setAscendingOrder(isAscendingOrder ? false : true);
```

Comment:
Since `isAscendingOrder` is a boolean value, we can simplify this line further without using conditional statement